### ISSUE TITLE
Cannot install deepchem==2.2.0

### ISSUE THREAD

**Issue by zhouhao-learning** [deepchem/deepchem#1626]
I tried to install the 2.2.0 version of deepchem, I executed the following installation command:
```
 conda install -c deepchem -c rdkit -c conda-forge -c omnia deepchem=2.2.0

```

But I can't install it properly, I get the following question:
```
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - deepchem=2.2.0

Current channels:

  - https://conda.anaconda.org/deepchem/linux-64
  - https://conda.anaconda.org/deepchem/noarch
  - https://conda.anaconda.org/rdkit/linux-64
  - https://conda.anaconda.org/rdkit/noarch
  - https://conda.anaconda.org/conda-forge/linux-64
  - https://conda.anaconda.org/conda-forge/noarch
  - https://conda.anaconda.org/omnia/linux-64
  - https://conda.anaconda.org/omnia/noarch
  - https://repo.anaconda.com/pkgs/main/linux-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/free/linux-64
  - https://repo.anaconda.com/pkgs/free/noarch
  - https://repo.anaconda.com/pkgs/r/linux-64
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://repo.anaconda.com/pkgs/pro/linux-64
  - https://repo.anaconda.com/pkgs/pro/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.

```

**Comment by VIGS25** [comment]
The conda build for `DeepChem 2.2.0` is not available yet (https://anaconda.org/deepchem/deepchem). If you want to use it, you'd have to install from source.

**Comment by peastman** [comment]
What do you mean?  It lists lots of 2.2.0 packages there.

@zhouhao-learning what OS and Python version are you using?  There are conda packages for Linux and Mac with Python 2.7, 3.6, and 3.7.

**Comment by zhouhao-learning** [comment]
@peastman 
The operating system I am using is Linux, and the python version is 3.6.

**Comment by peastman** [comment]
I don't know what's going on then.  `linux-64/deepchem-2.2.0-py36_0.tar.bz2` is one of the files listed at https://anaconda.org/deepchem/deepchem/files, so it should be finding it.  Also, the error message listed https://conda.anaconda.org/deepchem/linux-64 as one of the channels it was searching.

Perhaps it was just a temporary issue with Anaconda cloud?  Try installing again.  Does it still not work?

**Comment by gdreiman1** [comment]
Any update with this? I am having the same problems installing on Linux.

**Comment by DumbMachine** [comment]
The command `  conda install -c deepchem -c rdkit -c conda-forge -c omnia deepchem=2.2.0` doesn't work for me either. The following is the workaround I used to install deepchem:
1. Visit [here](https://anaconda.org/deepchem/deepchem/files) and download the source for your python version. ( In my case it was  deepchem-2.2.0-py37_0.tar.bz2 ).
2. Use this command to install with conda, ` conda install deepchem-2.2.0-py37_0.tar.bz2`.
3. Now if you have all the correct dependencies, this will work just fine.
4. If any error like `ModuleNotFoundError: No module named 'joblib'`, comes up then install the packages accordingly via conda.

**Comment by zhouhao-learning** [comment]
@DumbMachine 
Hello, I installed deepchem == 2.2.0 in my `anaconda'by your method, but when I checked the version of deepchem, I didn't get the result of 2.2.0.
```
>>> import deepchem as dc
/home/zh/sda3/Anaconda3/envs/deep_latest/lib/python3.6/site-packages/sklearn/ensemble/weight_boosting.py:29: DeprecationWarning: numpy.core.umath_tests is an internal NumPy module and should not be imported. It will be removed in a future NumPy release.
  from numpy.core.umath_tests import inner1d
>>> dc.__version__
'2.1.1'
>>> 

```

**Comment by peastman** [comment]
Sounds like you have two different copies of DeepChem on your computer, possibly in different Pythons or different environments.  Try this:

```python
import deepchem as dc
print(dc.__file__)
```

That will show you where the version you're currently using is located.

**Comment by VIGS25** [comment]
It looks as though the version number in `deepchem.__init__` file is not updated. It still says 2.1.1 there.

**Comment by peastman** [comment]
Oops!  Sorry about that!

**Comment by peastman** [comment]
Alright, I found the problem, and it was a totally silly one.  The 2.2.0 packages had been put under the label `deepchem` rather than `main`.  It should work now.  Give it a try and let me know if it works for you.

**Comment by marpat** [comment]
I installed DeepChem into a new Python 3.7 miniconda environment using conda like:
`conda install -c deepchem -c rdkit -c conda-forge -c omnia deepchem=2.2.0`
`conda list ` shows deepchem 2.2.0
`print(deepchem.__version__)` shows 2.1.1

Has the problem been fixed? "The 2.2.0 packages had been put under the label deepchem rather than main" - can you explain where had it been put or what to do differently to avoid the version mismatch?

**Comment by peastman** [comment]
>  print(deepchem.__version__) shows 2.1.1

Don't worry about that.  It just means we forgot to update that value before building 2.2.

> can you explain where had it been put or what to do differently to avoid the version mismatch?

I was talking about the files in the conda channel at https://anaconda.org/deepchem/deepchem/files.  Some of them had the wrong label, which prevented conda from finding them.  It's fixed now.
